agendas/2020-03-05.md

```diff
@@ -65,4 +66,5 @@ Example agenda item:
    - [Custom Scalar Specification URIs RFC Issue](https://github.com/graphql/graphql-spec/issues/635)
    - [Custom Scalar Specification URIs RFC PR](https://github.com/graphql/graphql-spec/pull/649)
    - [GraphQL.js PR](https://github.com/graphql/graphql-js/pull/2276/files)
+   - Question: should the spec suggest or require that implementations validate the URL is well-formed?
```

Comment:
👏 Thank you for adding this.

Question was raised by @chrisbutcher here: https://github.com/graphql/graphql-js/pull/2276#issuecomment-594598070